.gitconfig

```diff
@@ -0,0 +1,2 @@
+[core]
```

Comment:
I think we should accomplish this with a file at `tough/tests/data/.gitattributes` instead. The gitattributes file should declare that everything is binary, which I think will disable autocrlf.

Can you try it? I think the gitattributes file could look like this

`tough/tests/data/.gitattributes`

```text
# Denote all files as binary so they will not be modified by autocrlf.
*
```

The reason I think this is better is that it would not interfere with a Windows developer's preferred workflow.

Reply:
It works, commited in https://github.com/awslabs/tough/pull/342/commits/670fdf817b249ee126c1bf61cd6ffbe3150eb823